### PR TITLE
fix links and attributes on project readme pages

### PR DESCRIPTION
**Pull request by smccarthy-ie**
Cleans up project readme files - fixes broken links, adds missing links and attributes on:

- https://github.com/redhat-developer/app-services-guides
- https://github.com/redhat-developer/app-services-guides/blob/main/CONTRIBUTING.adoc

### CODE REVIEW

**Review by sterobin — APPROVED** [review]
Thanks @smccarthy-ie

File: README.adoc [hunk 1]
```diff
@@ -1,31 +1,39 @@
...
+= {product} guides and samples
 
+== {product-long-kafka} guides
 
-= {product} Guides and Samples
+* link:./docs/kafka/getting-started-kafka[Getting started with {product-long-kafka}]
+* link:./docs/kafka/rhoas-cli-getting-started-kafka[Getting started with the `rhoas` CLI for {product-long-kafka}]
+* link:./docs/kafka/quarkus-kafka[Using Quarkus applications with Kafka instances in {product-long-kafka}]
+* link:./docs/kafka/kcat-kafka[Using Kafkacat with Kafka instances in {product-long-kafka}]
+* link:./docs/kafka/kafka-bin-scripts-kafka[Using Kafka bin scripts with {product-long-kafka}]
```
Comment (bredamc) [inline README.adoc:13]:
```suggestion
* link:./docs/kafka/kafka-bin-scripts-kafka[Configuring and connecting Kafka scripts with {product-long-kafka}]
```

File: README.adoc [hunk 2]
```diff
@@ -1,31 +1,39 @@
...
 
+= {product} guides and samples
 
+== {product-long-kafka} guides
 
-= {product} Guides and Samples
+* link:./docs/kafka/getting-started-kafka[Getting started with {product-long-kafka}]
+* link:./docs/kafka/rhoas-cli-getting-started-kafka[Getting started with the `rhoas` CLI for {product-long-kafka}]
+* link:./docs/kafka/quarkus-kafka[Using Quarkus applications with Kafka instances in {product-long-kafka}]
+* link:./docs/kafka/kcat-kafka[Using Kafkacat with Kafka instances in {product-long-kafka}]
```
Comment (bredamc) [inline README.adoc:12]:
```suggestion
* link:./docs/kafka/kcat-kafka[Configuring and connecting Kafkacat with {product-long-kafka}]
```

File: README.adoc [hunk 3]
```diff
@@ -1,31 +1,39 @@
...
+:product-long-kafka: OpenShift Streams for Apache Kafka
+:product-long-registry: OpenShift Service Registry
 
+= {product} guides and samples
 
+== {product-long-kafka} guides
 
-= {product} Guides and Samples
+* link:./docs/kafka/getting-started-kafka[Getting started with {product-long-kafka}]
+* link:./docs/kafka/rhoas-cli-getting-started-kafka[Getting started with the `rhoas` CLI for {product-long-kafka}]
```
Comment (bredamc) [inline README.adoc:10]:
```suggestion
* link:./docs/kafka/rhoas-cli-getting-started-kafka[Getting started with the rhoas CLI for {product-long-kafka}]
```
Not sure about including command formatting in a doc title?

File: README.adoc [hunk 4]
```diff
@@ -1,31 +1,39 @@
...
-== Guides
+== {product-long-registry} guides
 
-* link:./docs/kafka/getting-started[Getting started with {product-kafka}]
-* link:./rhoas-cli[Installing and configuring the `rhoas` CLI]
-* link:./docs/service-discovery[Binding OpenShift applications to {product-kafka}]
+* link:./docs/registry/getting-started-registry/[Getting started with {product-long-registry}]
+* link:./docs/registry/access-mgmt-registry[Managing account access in {product-long-registry}]
+* link:./docs/registry/rhoas-cli-registry[Getting started with the `rhoas` CLI for {product-long-registry}]
+* link:./docs/registry/quarkus-registry[Using Quarkus applications with Kafka and {product-long-registry} instances]
```
Comment (bredamc) [inline README.adoc:26]:
```suggestion
* link:./docs/registry/quarkus-registry[Using Quarkus applications with Kafka instances and {product-long-registry} instances]
```

File: README.adoc [hunk 5]
```diff
@@ -1,31 +1,39 @@
...
 
-== Guides
+== {product-long-registry} guides
 
-* link:./docs/kafka/getting-started[Getting started with {product-kafka}]
-* link:./rhoas-cli[Installing and configuring the `rhoas` CLI]
-* link:./docs/service-discovery[Binding OpenShift applications to {product-kafka}]
+* link:./docs/registry/getting-started-registry/[Getting started with {product-long-registry}]
+* link:./docs/registry/access-mgmt-registry[Managing account access in {product-long-registry}]
+* link:./docs/registry/rhoas-cli-registry[Getting started with the `rhoas` CLI for {product-long-registry}]
```
Comment (bredamc) [inline README.adoc:25]:
```suggestion
* link:./docs/registry/rhoas-cli-registry[Getting started with the rhoas CLI for {product-long-registry}]
```
As before, not sure about using command formatting in a doc title.

File: README.adoc [hunk 6]
```diff
@@ -1,31 +1,39 @@
...
-* link:./docs/kafka/rhoas-cli-kafka[Getting started with the `rhoas` CLI for OpenShift Streams for Apache Kafka]
-* link:./docs/kafka/quarkus[Using Quarkus applications with Kafka instances in {product-kafka}]
-* link:./docs/kafka/kafkacat[Using Kafkacat with Kafka instances in {product-kafka}]
-* link:./docs/kafka/kafka-bin-scripts[Using Kafka bin scripts with {product-kafka}]
-
-== OpenShift Service Registry Guides
-
-* link:./docs/registry/rhoas-cli-service-registry[Getting started with the `rhoas` CLI for OpenShift Service Registry]
-* link:./docs/registry/getting-started-service-registry/[Getting started with Service Registry]
+* link:./docs/rhoas/rhoas-cli-installation[Installing and configuring the `rhoas` CLI]
```
Comment (bredamc) [inline README.adoc:30]:
```suggestion
* link:./docs/rhoas/rhoas-cli-installation[Installing and configuring the rhoas CLI]
```

File: README.adoc [hunk 7]
```diff
@@ -1,31 +1,39 @@
...
-* link:./docs/kafka/getting-started[Getting started with {product-kafka}]
-* link:./rhoas-cli[Installing and configuring the `rhoas` CLI]
-* link:./docs/service-discovery[Binding OpenShift applications to {product-kafka}]
+* link:./docs/registry/getting-started-registry/[Getting started with {product-long-registry}]
+* link:./docs/registry/access-mgmt-registry[Managing account access in {product-long-registry}]
+* link:./docs/registry/rhoas-cli-registry[Getting started with the `rhoas` CLI for {product-long-registry}]
+* link:./docs/registry/quarkus-registry[Using Quarkus applications with Kafka and {product-long-registry} instances]
 
-== OpenShift Streams For Apache Kafka Guides
+== Common guides
```
Comment (bredamc) [inline README.adoc:28]:
```suggestion
== Core guides
```
Maybe refer to them as Core rather than Common?

File: README.adoc [hunk 8]
```diff
@@ -1,31 +1,39 @@
...
-== OpenShift Service Registry Guides
-
-* link:./docs/registry/rhoas-cli-service-registry[Getting started with the `rhoas` CLI for OpenShift Service Registry]
-* link:./docs/registry/getting-started-service-registry/[Getting started with Service Registry]
+* link:./docs/rhoas/rhoas-cli-installation[Installing and configuring the `rhoas` CLI]
 
 == Application examples used in guides
 
 * link:./code-examples/quarkus-kafka-quickstart[Quarkus Kafka quick start]
+* link:./code-examples/quarkus-service-registry-quickstart[Quarkus Kafka Service Registry quick start]
```
Comment (bredamc) [inline README.adoc:35]: Does this work? I don't see a Readme.adoc in that directory?

File: README.adoc [hunk 9]
```diff
@@ -1,31 +1,39 @@
...
-
-== OpenShift Service Registry Guides
-
-* link:./docs/registry/rhoas-cli-service-registry[Getting started with the `rhoas` CLI for OpenShift Service Registry]
-* link:./docs/registry/getting-started-service-registry/[Getting started with Service Registry]
+* link:./docs/rhoas/rhoas-cli-installation[Installing and configuring the `rhoas` CLI]
 
 == Application examples used in guides
 
 * link:./code-examples/quarkus-kafka-quickstart[Quarkus Kafka quick start]
```
Comment (bredamc) [inline README.adoc:34]: There's another entry in the code-examples directory -- do you want to add this one too?:
* link:./code-examples/kafka-avro-schema-quickstart[Quarkus Kafka Avro Quick Start]

**Review by bredamc — COMMENTED** [review]
Just a few titles that are inaccurate, and a few questions :)

File: CONTRIBUTING.adoc [hunk 10]
```diff
@@ -41,7 +44,7 @@ npm --prefix .build run generate:attributes
 
 == Repository structure
 
-This repository is designed to be usable for people browsing it on GitHub, as well as reusable (e.g. in quick starts) in implementations of {product-kafka}.
+This repository is designed to be usable for people browsing it on GitHub, as well as reusable (e.g., in quick starts) in implementations of services such as {product-kafka} or {product-registry}.
```
Comment (bredamc) [inline CONTRIBUTING.adoc:47]:
```suggestion
This repository is designed to be usable for people browsing it on GitHub, as well as reusable (for example, in quick starts) in implementations of services such as {product-kafka} or {product-registry}.
```

File: CONTRIBUTING.adoc [hunk 11]
```diff
@@ -1,6 +1,9 @@
 :product: App Services
```
Comment (bredamc) [inline CONTRIBUTING.adoc:1]:
```suggestion
:product: Application Services
```
Not sure if this one should be spelled out in full?

File: README.adoc (hunk 8 above)
Comment (smccarthy-ie) [reply README.adoc:35]: Link should work, but there is no readme yet.

File: README.adoc [hunk 12]
```diff
@@ -1,31 +1,39 @@
 :product: App Services
```
Comment (bredamc) [inline README.adoc:1]:
```suggestion
:product: Application Services
```
For consistency, if you decided to make this change :)

File: README.adoc (hunk 9 above)
Comment (smccarthy-ie) [reply README.adoc:34]: Don't know much about that one, or how complete it is, can add a link later.

File: CONTRIBUTING.adoc (hunk 11 above)
Comment (smccarthy-ie) [reply CONTRIBUTING.adoc:1]: Don't think so. At least it has always been called App Services, deliberately AFAIK.

File: README.adoc [hunk 13]
```diff
@@ -1,31 +1,39 @@
...
+:product-long-kafka: OpenShift Streams for Apache Kafka
+:product-long-registry: OpenShift Service Registry
 
+= {product} guides and samples
 
+== {product-long-kafka} guides
 
-= {product} Guides and Samples
+* link:./docs/kafka/getting-started-kafka[Getting started with {product-long-kafka}]
```
Comment (bredamc) [inline README.adoc:9]: General comment for all the links that include attributes: I think you might want to remove the `link` keyword, or the attributes won't resolve in the URLs? (The links should still work though.)

Comment (smccarthy-ie) [reply README.adoc:9]: Thanks, yes wasn't sure about this one. Didn't see that behavior in the existing broken URLs. Will clean up in fresh PR if needed.

**Pull request merged**